### PR TITLE
Improved CommandHandler

### PR DESCRIPTION
**Pull request by evgfilim1**
Now you can pass list of commands instead of one command
It will be useful when you define some aliases for command or if commands (such as `/start` and `/help`) has the same behaviour

### CODE REVIEW

File: telegram/ext/commandhandler.py [hunk 1]
```diff
@@ -79,7 +80,10 @@ def __init__(self,
             pass_job_queue=pass_job_queue,
             pass_user_data=pass_user_data,
             pass_chat_data=pass_chat_data)
-        self.command = command
+        if isinstance(command, str):
```
Comment (jh0ker) [inline telegram/ext/commandhandler.py]: this could fail on python 2, if `command` is a `unicode` string

Comment (evgfilim1) [inline telegram/ext/commandhandler.py]: Fixed

**Comment by evgfilim1** [comment]
@jh0ker Fixed

**Comment by jh0ker** [comment]
Thanks :) I'll merge once the Travis tests are green

**Comment by evgfilim1** [comment]
Oh, I think I've broken this at all
```python
>>> _str = basestring
>>> _str
<type 'basestring'>
>>> '' is _str
False
>>> type('') == _str
False
>>> isinstance(type(''), _str)
False
>>> isinstance(type(u''), _str)
False
>>> type('')
<type 'str'>
>>> type(u'')
<type 'unicode'>
```

**Comment by Eldinnie** [comment]
you should try `isinstance("", _str)` and `isinstance(u"", _str)`

**Comment by evgfilim1** [comment]
@Eldinnie yes, that was my mistake, so I deleted message

**Pull request merged**